### ISSUE TITLE
rollback hangs if change other than last is selected

### ISSUE THREAD

**Issue by KramKroc** [onosproject/onos-config#950]
**Describe the bug**
When attempting a rollback which is not the latest change, the `onos config rollback` command just hangs.

**To Reproduce**
1. Make a couple of network changes so that there are a number of changes recorded, e.g. change1, change2, change3 where change3 is the newest change and change1 is the oldest
2. Run the command `onos config rollback change1` or `onos config rollback change2`
3. The command will simply hang

**Expected behavior**
Expect a warning to detail why the rollback is not possible

**Comment by SeanCondon** [comment]
While hanging is never good and needs to be fixed, I think there should not be an option to rollback anything other than the last Network Change. For this reason I don't think there should be any argument there for the change name - it should simply be `onos config rollback` or `onos config rollback-last-change` and the system figures out the latest

**Comment by KramKroc** [comment]
@SeanCondon agreed but I guess the act of supplying the name of the change being rolled back means a user has identified the change which needs rolled back and is making a concrete decision to rollback that change. I mean I'd worry in a multi-user scenario that two people could intend to rollback a change at roughly the same time, if they both simply issue `onos config rollback` then suddenly you've gone back two changes rather than rolling back the one change (and presumably getting an error for the second attempt by the second user).

**Comment by SeanCondon** [comment]
Thanks @KramKroc I can see that it is reassuring to know that only the change you want is rolled back.

If we were to require (or merely accept) a name on the rollback - we would return an error if this was not the latest or already had been rolled back. Do you think this is acceptable?

**Comment by KramKroc** [comment]
Sounds good to me.

**Comment by SeanCondon** [comment]
Fixed by #952 Thanks @Andrea-Campanella

**Comment by KramKroc** [comment]
Confirmed with latest code:

```
~ $ onos config get network-changes -v
CHANGE                          INDEX  REVISION  PHASE    STATE     REASON   MESSAGE
sad_liskov                      1       248      CHANGE   COMPLETE  NONE
	Device: devicesim1 (1.0.0)
	|/system/clock/config/timezone-name                |(STRING) Europe/Dublin                  |false  |


cocky_jepsen                    2       254      CHANGE   COMPLETE  NONE
	Device: devicesim1 (1.0.0)
	|/system/clock/config/timezone-name                |(STRING) Europe/Moscow                  |false  |


cocky_almeida                   3       260      CHANGE   COMPLETE  NONE
	Device: devicesim1 (1.0.0)
	|/system/clock/config/timezone-name                |(STRING) Europe/Rome                    |false  |


~ $ onos config rollback cocky_jepsen
rpc error: code = Unknown desc = change cocky_jepsen is not the last active on the stack of changes
~ $ onos config rollback sad_liskov
rpc error: code = Unknown desc = change sad_liskov is not the last active on the stack of changes
```
